README.md

```diff
@@ -100,6 +100,18 @@ npm start
 
 Go to http://localhost:3000 and you should see the sign in screen.
 
+### Running tests
+We use [Cypress](https://www.cypress.io/) to run tests, all the test files are under the ./cypress dir.
+
+```
+$(npm bin)/cypress open
+```
+or
+
+```
+node_modules/.bin/cypress open
+```
```

Comment:
We can probably add "cypress open" as a script in package.json so that it can be run cross-platform using "npm run integ-tests" or something like that.

Reply:
In that case I would call `cypress run`. Noted, will update the PR.